datasets/data_retrieval/mapping_broad.py

```diff
@@ -0,0 +1,373 @@
+                                             out_sr = list(feature_refs)[0])[0]
+    aoi_spatial_ref = aoi_projected.spatial_reference['wkid']
+    
+    for feature in feature_set.features:
+        feature_geometry = feature.geometry
+        region_label = feature.attributes['LABEL15']
+        feature_spatial_ref = feature_geometry['spatialReference']['wkid'] # WKID 102100, original WGS1984 Web Mercator projection
+        
+        # Get region category of the NJ Label:
+        for region in REGION_DICT.keys():
+            if region_label in REGION_DICT[region]:
+                region_cat = region
+                break
+
+        # Get intersection
+        intersection = intersect(feature_spatial_ref, 
+                                 [aoi_projected], 
+                                 feature_geometry)
+        
+        # Get square meter area of intersetion
+        if intersection:
+            area = areas_and_lengths(polygons = intersection[0], 
+                                    length_unit = 9001, # meters
+                                    area_unit = 9001, 
+                                    calculation_type = 'preserveShape', 
+                                    spatial_ref = aoi_spatial_ref)
+###
+    # area = arcgis.geometry.areas_and_lengths(polygons = intersection[0], 
+    #                                          length_unit= 9001, 
+    #                                          area_unit=9001, 
+    #                                          calculation_type='geodesic', 
+    #                                          spatial_ref = spatial_ref)
+###            
+
+
+        # Add row to DataFrame
+        feature_areas.loc[len(feature_areas)] = {'Region_Label': region_label, 
+                                                 'Region': region_cat,
+                                                 'Intersect_Area': area['areas'][0],
+                                                 'Region_Intersect_Total': 0}
+
+    return feature_areas
+
+def get_landcover_percentages(feature_areas):
+    '''
+    Function to determine percentage of area of interest covered by each land cover category (Grassy, Dense Wood, Woody, Suburban, Watery, Urban).
+    
+    Parameters:
+    -----------
+    layer - ArcGIS map layer to query.
+    lat - Float. Latitude to center on.
+    lon - Float. Longitude to center on.
+    radius - Float. Fraction of a mile to use as side length for area of interest square.
+    aoi - Area of interest. ArcGIS Polygon object.
+    feature_areas - DataFrame containing regions and the area of their intersection with the given area of interest (lat/lon).
+
+    Returns:
+    --------
+    region_percents - Dictionary of percentage of land in area of interest covered by each region category (index: Grassy, Dense Wood, Woody, Suburban, Watery, Urban).
+
+    '''
+    #aoi = get_area_of_interest(lat, lon, radius)
+    #feature_areas = get_aoi_feature_areas(layer, aoi)

```

Comment:
Another piece of commented out code.

Reply:
Removed, and adjusted function parameter description to reflect the removal of these lines.